tests/unit/unit-util.cpp

```diff
@@ -229,6 +229,41 @@ TEST_P(UnitUtilDimacsTests, cnf_to_dimacs)
 }
 
 
+
+TEST_P(UnitUtilDimacsTests, tseitin)
+{
+  Term p = s->make_symbol("p", boolsort);
+  Term q = s->make_symbol("q", boolsort);
+  Term r = s->make_symbol("r", boolsort);
+  Term t = s->make_symbol("t", boolsort);
+  if (s->get_solver_enum() == BTOR || s->get_solver_enum() == BZLA)
```

Comment:
These are the two solvers that alias booleans and bit-vectors of size one just FYI. Also, if incremental solving is an issue, you can enable it with: `s->set_opt("incremental", "true")`.

Reply:
Thanks!